src/style.css

```diff
@@ -0,0 +1,133 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
+  display: flex;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.card__photo {
+  margin-top: 20px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__description {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: flex-start;
+  width: 100%;
+  margin: 0 14px;
+}
+
+.card__name {
+  display: flex;
+  flex-direction: column;
+  margin-top: 26px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.card__code {
+  display: flex;
+  margin-top: 6px;
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 13px;
+}
+
+.card__reviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin: 0 15px ;
+}
+
+.card__stars_active {
+  display: flex;
+
+}
+
+.card__star_active {
+  box-sizing: border-box;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  height: 15px;
+  width: 15px;
+}
+
+.card__star_active:last-child {
+  background-image: url(images/star.svg);
+}
+
+.card__reviews_comments {
+  margin: 0;
+  font-size: 11px;
+  font-weight: 400;
+}
+
+.card__primary {
+  margin: 0 15px;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+
+}
+
+.card__primary_price {
+  margin: 0;
+  font-size: 11px;
+  font-weight: 400;
+  line-height: 13px;
+}
+
+.card__primary_value {
+  margin: 0;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 19px;
+}
+
+.card__button_active {
+  display: flex;
+  width: 170px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  text-decoration: none;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16;
+  color: #fff;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 6px;
+}
+
+.card__button_active:hover {
+  color: #00acdc;
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
at this moment when you hovered on your link - the border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change its position, so it's better if you write border property inside the button.